ui/admin/app/controllers/scopes/scope/host-catalogs/host-catalog/host-sets/index.js

```diff
@@ -74,8 +75,24 @@ export default class ScopesScopeHostCatalogsHostCatalogHostSetsIndexController e
     isNew ? 'notifications.create-success' : 'notifications.save-success',
   )
   async save(hostSet) {
+    // Fetch newest host set as updates to host set attributes cause an async db update which
+    // updates the version again and can cause a version mismatch if the host set is updated
+    // again and we haven't fetched the newest version.
```

Comment:
See this [thread](https://hashicorp.slack.com/archives/C07E8C15JGL/p1736378911635939) for more info